jina/serve/runtimes/worker/__init__.py

```diff
@@ -49,11 +49,30 @@ async def async_setup(self):
                     namespace='jina',
                     labelnames=('runtime_name',),
                 )
-                    .labels(self.args.name)
-                    .time()
+                .labels(self.args.name)
+                .time()
             )
+
+            self._failed_requests_metrics = Gauge(
```

Comment:
Same: I believe this should be a Counter and not a Gauge since the number cannot decrease